README.md

```diff
@@ -1,30 +1,20 @@
-compliance
-==========
+# CTK
+An extensible Conformance Test Kit for the GA4GH Server
```

Comment:
Change 'An extensible Conformance Test Kit for the GA4GH Server' to 'An extensible Conformance Test Kit for the GA4GH API' ?

The aim is to have more than one server...